AdornUI-documentation/alert.html

```diff
@@ -0,0 +1,115 @@
+                <a class="sidebar-item-link" href="alert.html">Alerts</a>
+                <a class="sidebar-item-link" href="badges.html">Badges</a>
+                <a class="sidebar-item-link" href="avatar.html">Avatar</a>
+                <a class="sidebar-item-link" href="card.html">Cards</a>
+                <a class="sidebar-item-link" href="button.html">Buttons</a>
+                <a class="sidebar-item-link" href="input.html">Input</a>
+                <a class="sidebar-item-link" href="navigation.html">Navigation</a>
+            </div>
+        </aside>
+        <main>
+            <section class="component-doc-container">
+                <h2 class="component-title">Alerts</h2>
+                <h5 class="component-desc"> Alerts are used to force the user to read a specific message which
+                    might specify danger,success,warning or certain information to the
+                    user. It takes away the attention of the user from current window and
+                    diverts him to specific message.<br></br>
+                    To use this alert, add class <span class="keyword-highlighter">alert-box</span>. Include classes
+                    <span class="keyword-highlighter">alert-success</span>,
+                    <span class="keyword-highlighter">alert-danger</span>,<span
+                        class="keyword-highlighter">alert-warning</span>,<span
+                        class="keyword-highlighter">alert-primary</span>,<span
+                        class="keyword-highlighter">alert-secondary</span>
+                </h5>
+                <div class="component-container">
+                    <div class="alert-box alert-success">
+                        <span><i class="fa fa-check-circle"></i> Success! Indicates positive action to the user.</span>
+                    </div>
+                    <div class="alert-box alert-danger">
+                        <span><i class="fa fa-exclamation-triangle"></i> Danger! Indicates danger or negative action to
+                            the
+                            user.</span>
+                    </div>
+                    <div class="alert-box alert-warning">
+                        <span> <i class="fa fa-exclamation-triangle"></i> Warning Alert. Indicates warning to the user.
+                        </span>
+                    </div>
+                    <div class="alert-box alert-primary">
+                        Primary Alert. Indicates any regular or normal information to the
+                        user.
+                    </div>
+                    <div class="alert-box alert-secondary">
+                        Secondary Alert. Indicates any secondary information to the user. </div>
+                </div>
+                <iframe src="https://www.thiscodeworks.com/embed/61f9326e1e1df40015379400"
+                    style="width: 90%; height: 300px;" frameborder="0"></iframe>
+                <h3 class="component-subtitle">
```

Comment:
you can make one class for code snippets and use it everywhere instead of using inline styles.